common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -6662,6 +6720,15 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=9016)
     public static final String A_zimbraFeatureDocumentEditingEnabled = "zimbraFeatureDocumentEditingEnabled";
 
+    /**
+     * Whether to show &quot;Go to URL&quot; in context menu when an email
+     * link or an address bubble is right-clicked.
+     *
+     * @since ZCS 11.0.0
+     */
+    @ZAttr(id=9005)
+    public static final String A_zimbraFeatureEmailGoToURLEnabled = "zimbraFeatureEmailGoToURLEnabled";
```

Comment:
can't this functionality achieved using zimlet instead of adding ldap attribute?

Reply:
It has been used on our customer with customization. It needs to be added with "id=9005" to avoid conflict in the future.
Let me keep it as is for now. I will check if we can move the function to a zimlet easily. If it can be achieved, I will remove the attribute and create another PR.

Reply:
created ZCS-12545